<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinListitem</class>
 <widget class="QWidget" name="BinListitem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1360</width>
     <height>158</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);border-radius:15px;</string>
   </property>
   <widget class="PictureBox" name="thumbnail" native="true">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>30</y>
      <width>176</width>
      <height>99</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_title">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>48</y>
      <width>214</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(26, 26, 26, 1)</string>
    </property>
    <property name="text">
     <string>两分钟上手会议瑞记</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_duration">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>91</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(102, 102, 102, 1)</string>
    </property>
    <property name="text">
     <string>2分16秒</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_createTime">
    <property name="geometry">
     <rect>
      <x>827</x>
      <y>68</y>
      <width>281</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(102, 102, 102, 1)</string>
    </property>
    <property name="text">
     <string>2022年8月15日 上午8:55</string>
    </property>
   </widget>
   <widget class="PictureBox" name="trash_pic" native="true">
    <property name="geometry">
     <rect>
      <x>1301</x>
      <y>67</y>
      <width>38</width>
      <height>38</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="id">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>20</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>74</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="PictureBox" name="recover_pic" native="true">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>67</y>
      <width>38</width>
      <height>38</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureBox</class>
   <extends>QWidget</extends>
   <header>picturebox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
